README.adoc

```diff
@@ -157,18 +157,18 @@ have IPv4 addresses, the address on eth2 is used.
 === HTTP Proxy Settings
 
 In an environment where the HTTP traffic needs to pass through an
-HTTP proxy server, set the `http_proxy`, `http_proxy_user` and
-`http_proxy_password` proxy configurations in the Vagrantfiles to enable
+HTTP proxy server, set the `proxy`, `proxy_user` and
```

Comment:
> One also need to thing about introducing something like a migration guide which would document these type of changes between releases.

@hferentschik Agree. Basically it should contain all changes that are not backward compatible.